### PR TITLE
Content migration batch 4

### PR DESCRIPTION
**Pull request by mattletford**
Migrating content

From: https://developers.google.com/web/fundamentals/media/recording-audio
To: https://web.dev/media-recording-audio

From: https://developers.google.com/web/fundamentals/media/recording-video
To: https://web.dev/media-recording-video

From: https://developers.google.com/web/fundamentals/native-hardware/click-to-call
To: https://web.dev/native-hardware-click-to-call

From: https://developers.google.com/web/fundamentals/native-hardware/device-orientation
To: https://web.dev/native-hardware-device-orientation

From: https://developers.google.com/web/fundamentals/native-hardware/fullscreen
To: https://web.dev/native-hardware-fullscreen

@rachelandrew

### CODE REVIEW

File: src/site/content/en/blog/native-hardware-click-to-call/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+authors:
+  - petelepage
+date: 2014-06-17
+updated: 2019-07-26
+---
+
+On devices with phone capabilities, make it easy for users to directly connect
+with you by simply tapping a phone number, more commonly known as click to call.
+
+## TL;DR {: .hide-from-toc }
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-click-to-call/index.md:15]:
```suggestion
## TL;DR
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+Device motion and orientation events provide access to the built-in
+accelerometer, gyroscope, and compass in mobile devices.
+
+These events can be used for many purposes; in gaming, for example, to control
+the direction or action of a character. When used with geolocation, they can
+help create more accurate turn-by-turn navigation or provide information about
+a specific location.
+
+{% Aside 'caution' %}
+Caution: Not all browsers use the same coordinate system, and they may report different values under identical situations. This has improved over time, but be sure to test your situation.
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:21]:
```suggestion
Not all browsers use the same coordinate system, and they may report different values under identical situations. This has improved over time, but be sure to test your situation.
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+
+To use the data that the device orientation and motion events return,
+it is important to understand the values provided.
+
+### Earth coordinate frame
+
+The Earth coordinate frame, described by the values `X`, `Y`, and `Z`, is aligned
+based on gravity and standard magnetic orientation.
+
+<div class="table-wrapper scrollbar">
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:39]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+  <td>Represents the north-south direction (where north is positive).</td>
+</tr>
+<tr>
+  <td><code>Z</code></td>
+  <td>Represents the up-down direction, perpendicular to the ground
+      (where up is positive).
+  </td>
+</tr>
+</table>
+</div>
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:57]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+    </figcaption>
+</figure>
+
+<!-- Special thanks to Sheppy (https://developer.mozilla.org/en-US/profiles/Sheppy)
+  for his images which are in the public domain. -->
+
+The device coordinate frame, described by the values `x`, `y`, and `z`, is aligned
+based on the center of the device.
+
+<div class="table-wrapper scrollbar">
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:74]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+    <td>In the plane of the screen, positive towards the top.</td>
+    </tr>
+    <tr>
+    <td><code>Z</code></td>
+    <td>Perpendicular to the screen or keyboard, positive extending
+        away.
+    </td>
+    </tr>
+</table>
+</div>
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:92]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+    <figcaption>
+      Illustration of gamma in the device coordinate frame
+    </figcaption>
+  </figure>
+
+The rotation around the y axis. The `gamma` value is 0&deg; when the left and
+right edges of the device are equidistant from the surface of the earth. The value
+increases as the right side is tipped towards the surface of the earth.
+
+<div style="clear:both;"></div>
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:144]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+
+The event returns four properties,
+[`accelerationIncludingGravity`](#device-coordinate-frame),
+[`acceleration`](#device-coordinate-frame), which excludes the effects of
+gravity, [`rotationRate`](#rotation-data), and `interval`.
+
+For example, let's take a look at a phone, lying on a flat table, with its
+screen facing up.
+
+<div class="table-wrapper scrollbar">
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:234]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 9]
```diff
@@ -0,0 +1,337 @@
...
+    </tr>
+    <tr>
+      <td data-th="State">Moving up and to the right</td>
+      <td data-th="Rotation">[0, 0, 0]</td>
+      <td data-th="Acceleration">[5, 0, 5]</td>
+      <td data-th="Acceleration with gravity">[5, 0, 14.81]</td>
+    </tr>
+  </tbody>
+</table>
+</div>
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:271]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 10]
```diff
@@ -0,0 +1,337 @@
...
+
+Conversely, if the phone were held so the screen was perpendicular to the
+ground, and was directly visible to the viewer:
+
+<div class="table-wrapper scrollbar">
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:276]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-device-orientation/index.md [hunk 11]
```diff
@@ -0,0 +1,337 @@
...
+    </tr>
+    <tr>
+      <td data-th="State">Moving up and to the right</td>
+      <td data-th="Rotation">[0, 0, 0]</td>
+      <td data-th="Acceleration">[5, 5, 0]</td>
+      <td data-th="Acceleration with gravity">[5, 14.81, 0]</td>
+    </tr>
+  </tbody>
+</table>
+</div>
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-device-orientation/index.md:313]:
```suggestion
```

File: src/site/content/en/blog/native-hardware-fullscreen/index.md [hunk 12]
```diff
@@ -0,0 +1,565 @@
...
+
+- `element.requestFullscreen()` (currently prefixed in Chrome, Firefox, and IE)
+  displays the element in fullscreen mode.
+- `document.exitFullscreen()` (currently prefixed in Chrome, Firefox and IE.
+  Firefox uses `cancelFullScreen()` instead) cancels fullscreen mode.
+- `document.fullscreenElement` (currently prefixed in Chrome, Firefox, and IE)
+  returns true if any of the elements are in fullscreen mode.
+
+{% Aside %}
+Note: You will notice that in the prefixed versions there is a lot of
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-fullscreen/index.md:41]:
```suggestion
You will notice that in the prefixed versions there is a lot of
```

File: src/site/content/en/blog/native-hardware-fullscreen/index.md [hunk 13]
```diff
@@ -0,0 +1,565 @@
...
+  );
+</script>
+```
+
+The above example is a little contrived; I've hidden all the complexity around
+the use of vendor prefixes.
+
+{% Aside %}
+Note: Damn you, vendor prefixes!
+{% endAside %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-fullscreen/index.md:73]:
```suggestion
```

File: src/site/content/en/blog/media-recording-audio/index.md [hunk 14]
```diff
@@ -0,0 +1,265 @@
...
+
+Users hate getting prompted for access to powerful devices on their machine and
+they will frequently block the request, or they will ignore it if they don't
+understand the context of which the prompt has been created. It is best practice
+to only ask to access the microphone when first needed. Once the user has
+granted access, they won't be asked again, however, if they reject access,
+you can't ask the user for permission again.
+
+{% Aside 'warning' %}
+Warning: Asking for access to the microphone on page load will result in most of your users
```
Comment (rachelandrew) [inline src/site/content/en/blog/media-recording-audio/index.md:228]:
```suggestion
Asking for access to the microphone on page load will result in most of your users
```

File: src/site/content/en/blog/native-hardware-fullscreen/index.md [hunk 15]
```diff
@@ -0,0 +1,565 @@
...
+```html
+<meta name="apple-mobile-web-app-capable" content="yes" />
+```
+
+> If content is set to yes, the web application runs in full-screen mode;
+> otherwise, it does not. The default behavior is to use Safari to display web
+> content. You can determine whether a webpage is displayed in full-screen mode
+> using the window.navigator.standalone read-only Boolean JavaScript property.
+> <a href="https://developer.apple.com/library/safari/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/MetaTags.html">Apple</a>
+
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-fullscreen/index.md:243]:
```suggestion
<blockquote>
<p>If content is set to yes, the web application runs in full-screen mode; otherwise, it does not. The default behavior is to use Safari to display web content. You can determine whether a webpage is displayed in full-screen mode using the window.navigator.standalone read-only Boolean JavaScript property.</p>
<cite><a href="https://developer.apple.com/library/safari/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/MetaTags.html">Apple</a></cite>
</blockquote>


```

Comment (rachelandrew) [reply src/site/content/en/blog/native-hardware-fullscreen/index.md:243]: https://web.dev/handbook/web-dev-components/#block-quotes

File: src/site/content/en/blog/native-hardware-fullscreen/index.md [hunk 16]
```diff
@@ -0,0 +1,565 @@
...
+similar to the iOS Safari model.
+
+```html
+<meta name="mobile-web-app-capable" content="yes" />
+```
+
+> You can set up your web app to have an application shortcut icon added to a
+> device's home screen, and have the app launch in full-screen "app mode" using
+> Chrome for Android's "Add to Home screen" menu item.
+> <a href="https://developers.chrome.com/multidevice/android/installtohomescreen">Google Chrome</a>
```
Comment (rachelandrew) [inline src/site/content/en/blog/native-hardware-fullscreen/index.md:257]:
```suggestion
<blockquote>
<p>You can set up your web app to have an application shortcut icon added to a device's home screen, and have the app launch in full-screen "app mode" using Chrome for Android's "Add to Home screen" menu item.</p>
<cite><a href="https://developers.chrome.com/multidevice/android/installtohomescreen">Google Chrome</a></cite>
</blockquote>
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
All my changes are suggestions so please approve these as a batch from the UI so we can see they are closed.

File: src/site/content/en/blog/native-hardware-fullscreen/index.md (hunk 16 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-fullscreen/index.md]: @rachelandrew added blockquote

File: src/site/content/en/blog/native-hardware-fullscreen/index.md (hunk 15 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-fullscreen/index.md]: @rachelandrew added blockquote

File: src/site/content/en/blog/media-recording-audio/index.md (hunk 14 above)
Comment (mattletford) [reply src/site/content/en/blog/media-recording-audio/index.md]: @rachelandrew removed

File: src/site/content/en/blog/native-hardware-fullscreen/index.md (hunk 13 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-fullscreen/index.md]: @rachelandrew removed

File: src/site/content/en/blog/native-hardware-fullscreen/index.md (hunk 12 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-fullscreen/index.md]: @rachelandrew removed

File: src/site/content/en/blog/native-hardware-device-orientation/index.md (hunk 5 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-device-orientation/index.md]: @rachelandrew removed

File: src/site/content/en/blog/native-hardware-device-orientation/index.md (hunk 4 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-device-orientation/index.md]: @rachelandrew removed

File: src/site/content/en/blog/native-hardware-device-orientation/index.md (hunk 3 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-device-orientation/index.md]: @rachelandrew removed

File: src/site/content/en/blog/native-hardware-device-orientation/index.md (hunk 2 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-device-orientation/index.md]: @rachelandrew removed

File: src/site/content/en/blog/native-hardware-click-to-call/index.md (hunk 1 above)
Comment (mattletford) [reply src/site/content/en/blog/native-hardware-click-to-call/index.md]: @rachelandrew removed

**Review by rachelandrew — APPROVED** [review]
